invisible_cities/config/beersheba.conf

```diff
@@ -17,6 +17,7 @@ same_peak = True
 deconv_params = dict(
   q_cut         = 10,
   drop_dist     = [10., 10.],
+  cluster_size  = 1,
```

Comment:
if 2d, this parameter should not be given... Unless the 2d case can be generalized to work with different cluster sizes.